shaders/src/common_graphics.fs

```diff
@@ -97,3 +97,19 @@ vec3 heatmap(float v) {
     vec3 r = v * 2.1 - vec3(1.8, 1.14, 0.3);
     return 1.0 - r * r;
 }
+
+vec3 uintToColorDebug(uint v) {
```

Comment:
there is already some code that does that in `tone_mapping.fs` maybe this could be moved here.

Reply:
`Tonemap_DisplayRange`? Hmm, it seems like it's written specifically for tone mapping, so I'm not sure it makes sense to move it into common_graphics.

Reply:
It's `log2` based but the only line that's display range specific is: `float v = log2(luminance(x) / 0.18);`. We could make a more generic variant. I used to use it for other things as well. Now I often just use `heatmap()`.

Reply:
I'm all for refactoring it into a generic function, but I don't think cascade visualization is a good use case. Consecutive colors inside `Tonemap_DisplayRange` are too visually similar; we want cascade steps to be very discrete.